user_modules/shared_testcases/testCasesForPolicyTable.lua

```diff
@@ -824,5 +834,230 @@ function testCasesForPolicyTable:Restore_preloaded_pt()
 		commonPreconditions:RestoreFile(pt_fileName)
 	end
 end	
-	
+
+-- The funcion will be used when PTU is triggered. 
+-- 1. It is assumed that notification is recevied: EXPECT_HMINOTIFICATION("SDL.OnStatusUpdate", {status = "UPDATE_NEEDED"})
+-- 2. It is assumed that request/response is received: EXPECT_HMICALL("BasicCommunication.PolicyUpdate", 
+-- 3. Function will use default endpoints
+function testCasesForPolicyTable:flow_PTU_SUCCEESS_EXTERNAL_PROPRIETARY(app_id, device_id, hmi_app_id, ptu_file_path, ptu_file_name)
+	function Test:TestStep_Flow_PTU_SUCCEESS_EXTERNAL_PROPRIETARY()
+		if (app_id == nil) then app_id = config.application1.registerAppInterfaceParams.appID end
+		if (device_id == nil) then device_id = config.deviceMAC end
+		if (hmi_app_id == nil) then hmi_app_id = self.applications[config.application1.registerAppInterfaceParams.appName] end 
+		if (ptu_file_path == nil) then ptu_file_path = "files/" end
+		if (ptu_file_name == nil) then ptu_file_name = "PolicyTableUpdate" end
+		local ptu_file = ptu_file_path.."ptu.json"
+
+		--[[Start get data from PTS]]
+		--TODO(istoimenova): function for reading INI file should be implemented
+        --local SystemFilesPath = commonSteps:get_data_form_SDL_ini("SystemFilesPath")
+        local SystemFilesPath = "/tmp/fs/mp/images/ivsu_cache/"
+        local file_pts = SystemFilesPath.."sdl_snapshot.json"
+
+		-- Check SDL snapshot is created correctly and get needed data 
+	    testCasesForPolicyTableSnapshot:create_PTS(true, {app_id}, {device_id}, {hmi_app_id})
+
+        --BasicCommunication.PolicyUpdate is sent immediatelly at triggering PTU. Should be checked in the scripts
+        -- EXPECT_HMICALL("BasicCommunication.PolicyUpdate", { file = file_pts, timeout = timeout_after_x_seconds, retry = seconds_between_retries})
+        -- :Do(function(_,data) self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {}) end)
+
+        local endpoints = {}
+        local is_app_esxist = false
+
+        for i = 1, #testCasesForPolicyTableSnapshot.pts_endpoints do
+          --using default URLs
+          if (testCasesForPolicyTableSnapshot.pts_endpoints[i].service == "0x07") then
+            endpoints[1] = { url = testCasesForPolicyTableSnapshot.pts_endpoints[i].value, appID = nil}
+          end
+        end
```

Comment:
@Itileda According to APPLINK-19133 and as usual we have registered application, the endpoints should include also urls for appID(hmi).
 SNAPSHOT (hardcoded BTW) - It is not hardcoded. The hardcoded are only elements described in DataDictionary. According to these harccoded parameters sdl_snapshot is checked additionally(prints in console) and after that all include in endpoints section for service 0x07: default, appID should be used.